### ISSUE TITLE
show world coordinates in status bar

### ISSUE THREAD

**Issue by HoraK-FDF** [mapeditor/tiled#3096]
i think that would help with working on complete worlds too.

**Comment by bjorn** [comment]
You mean always or only when using the World Tool? And in pixels or in tiles?

I'm afraid due to the many options it could get a bit much (local tile coordinates, local pixel coordinates, world pixel coordinates, ...), and to have them all as text in the status bar would not be ideal, so potentially we could make a nicer representation and add a few toggles.

**Comment by HoraK-FDF** [comment]
Sorry wanted to write that but forgot it. Yes only when World Tool is activated and in tiles.

**Comment by eishiya** [comment]
> in tiles.

What does "in tiles" really mean in the context of the World? Different maps may have different tile sizes, and maps may be positioned off any tile grid.

**Comment by HoraK-FDF** [comment]
ok didn't take this into consideration that they could have different tile sizes, in this case maybe the smallest tile (or if tile is not a square the shortest side of the square) in the world should define the the size of the grid.

**Comment by bjorn** [comment]
Of course, most of the time they will be the same, and possibly it will suffice to use the grid size of the currently active map.

**Comment by eishiya** [comment]
I think using the tile size of the current map sounds good.

However, since these coordinates may be rounded since the maps are placed with sub-tile (pixel) precision, I think it would be good to make their inexactness explicit so that users know not to rely on these coordinates, e.g. by making the display read "(approx. 5462, 20)" rather than just the coordinates.
